cfg/rke2-cis-1.24/node.yaml

```diff
@@ -432,14 +432,14 @@ groups:
             - flag: RotateKubeletServerCertificate
               path: '{.featureGates.RotateKubeletServerCertificate}'
               set: false
+        type: skip
```

Comment:
I can see that check 1.3.6 isn't applicable for RKE2: [the docs](https://ranchermanager.docs.rancher.com/v2.7/reference-guides/rancher-security/hardening-guides/rke2-hardening-guide/rke2-self-assessment-guide-with-cis-v1.24-k8s-v1.24#136-ensure-that-the-rotatekubeletservercertificate-argument-is-set-to-true-automated)

but 4.2.12 is manual check and should pass: [4.2.12 Verify that the RotateKubeletServerCertificate argument is set to true (Manual)](https://ranchermanager.docs.rancher.com/v2.7/reference-guides/rancher-security/hardening-guides/rke2-hardening-guide/rke2-self-assessment-guide-with-cis-v1.24-k8s-v1.24#4212-verify-that-the-rotatekubeletservercertificate-argument-is-set-to-true-manual)

maybe we shouldn't skip it?

Reply:
4.2.12 is automated here too https://docs.rke2.io/security/cis_self_assessment124#4212-verify-that-the-rotatekubeletservercertificate-argument-is-set-to-true-automated

Reply:
and 1.3.6 is automated here...
https://docs.rke2.io/security/cis_self_assessment124#136-ensure-that-the-rotatekubeletservercertificate-argument-is-set-to-true-automated